salt/modules/win_pkg.py

```diff
@@ -581,9 +581,22 @@ def _refresh_db_conditional(saltenv, **kwargs):
 
 def refresh_db(**kwargs):
     '''
-    Fetches metadata files and calls :py:func:`pkg.genrepo
+    Fetches metadata files from the winrepo_dir and calls :py:func:`pkg.genrepo
     <salt.modules.win_pkg.genrepo>` to compile updated repository metadata.
 
+    The default winrepo directory on the master is `/srv/salt/win/repo-ng`. All
+    files that end with `.sls` in this and all subdirectories will be used to
+    generate the repository metadata (`winrepo.p`).
+
+    .. note::
+        Hidden directories (directories beginning with `.`, such as `.git`) will
```

Comment:
Can you do dot points under notes ?

*Hidden directories (directories beginning with `.`, such as `.git`) will not be fetched.
*A result with a `total: 0`  general means the files are located on master in the the wrong location. Try running `salt-call -l debug pkg.refresh saltenv=base` on the minion.